examples/arithmetics/src/language-server/main-browser.ts

```diff
@@ -26,14 +28,20 @@ const documentChangeNotification = new NotificationType<DocumentChange>('browser
 const jsonSerializer = arithmetics.serializer.JsonSerializer;
 shared.workspace.DocumentBuilder.onBuildPhase(DocumentState.Validated, documents => {
     for (const document of documents) {
-        const json = jsonSerializer.serialize(document.parseResult.value, {
-            sourceText: true,
-            textRegions: true,
-        });
+        const json = [];
+        const module = document.parseResult.value as Module;
+        if(document.diagnostics === null || document.diagnostics!.filter((i) => i.severity === 1).length === 0) {
+            for (const [evaluation, value] of interpretEvaluations(module)) {
+                const cstNode = evaluation.expression.$cstNode;
+                if (cstNode) {
+                    json.push({range: evaluation.expression.$cstNode?.range, text: evaluation.expression.$cstNode?.text, value: value});
+                }
+            }
+        }
         connection.sendNotification(documentChangeNotification, {
             uri: document.uri.toString(),
-            content: json,
+            content: JSON.stringify({ ast: jsonSerializer.serialize(module), evaluations: json }),
```

Comment:
@emilkrebs I have the impression you added the above mentioned config objects in the `domainmodel` example (which might also be helpful someday), but I need it here in line 43